tests/integration-tests/utils.py

```diff
@@ -208,6 +210,7 @@ def delete_s3_bucket(bucket_name, region):
     try:
         bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
         bucket.objects.all().delete()
+        bucket.object_versions.all().delete()
```

Comment:
Needed because we're turning on versioning in `create_s3_bucket` in the same file?